Scripts/scripts/cgal_create_cmake_script

```diff
@@ -58,6 +58,10 @@ if ( CGAL_FOUND )
   include( CGAL_CreateSingleSourceCGALProgram )
 
 EOF
+    if [ -d "${SOURCE_DIR}" ] ; then
+      echo "  set(CGAL_CURRENT_SOURCE_DIR \"${SOURCE_DIR}\")"
+      echo
+    fi
```

Comment:
If the sources already have a `CMakeLists.txt`, then the `CMAKE_CURRENT_SOURCE_DIR` is the right one, and the line
```CMake
set(CGAL_CURRENT_SOURCE_DIR ${CMAKE_CURRENT_SOURCE_DIR})
```
[in `UseCGAL.cmake`][line] will set the variable.

[line]: https://github.com/CGAL/cgal/pull/1357/files/ce782f02f8983669678f78ace85dd6a13760cece#diff-bbff21e986dce690cf536e0de94bc0f8R12